test/release_plugin_test.js

```diff
@@ -115,4 +114,45 @@ exports.release_plugin_snapshot = {
             test.done();
         });
     }
-};
\ No newline at end of file
+};
+
+exports.release_plugin_snapshot_no_tag = {
+    setUp: function (done) {
+        setUpTests('', done);
+    },
+
+    tearDown: tearDownTests,
+
+    currentVersion: function (test) {
+        test.expect(1);
+
+        callGrunt('gruntfile.js', 'currentVersion', function (error, stdout) {
+            var expected = '{"currentVersion":"0.0.1-SNAPSHOT"}',
+            actual = getJsonFromOutput(stdout);
+
+            test.equal(actual, expected, 'should return current snapshot project version');
+            test.done();
+        });
+    },
+
+    metadata: function (test) {
+        test.expect(1);
+
+        callGrunt('gruntfile.js', 'metadata', function (error, stdout) {
+            var expected = '{"version":"0.0.1-SNAPSHOT","name":"some-name","domain":"some-domain"}',
+            actual = getJsonFromOutput(stdout);
+
+            test.equal(actual, expected, 'should return project metadata with snapshot version');
```

Comment:
I suggest changing description to: 'should return project metadata with "0.0.1-SNAPSHOT" version'